doc/whats_new.rst

```diff
@@ -1,3 +1,14 @@
+0.2.3
+=====
+
+Changelog
+---------
+
+New features
+............
+    - Mathematical formulas based on numpy functions can be applied on an
+    image or a list of images using :func:`nilearn.image.math_img`.
```

Comment:
You are going to have an rst error: the indentation here is incorrect (the line after the "-" should start below the "M" of "Mathematical")